presto-hive/src/main/java/io/prestosql/plugin/hive/metastore/glue/GlueHiveMetastore.java

```diff
@@ -536,6 +536,33 @@ public void setDatabaseOwner(HiveIdentity identity, String databaseName, HivePri
         throw new PrestoException(NOT_SUPPORTED, "setting the database owner is not supported by Glue");
     }
 
+    @Override
+    public void setTableOwner(HiveIdentity identity, String databaseName, String tableName, HivePrincipal principal)
```

Comment:
How can we have a test for that?

Reply:
I made one in AbstractHiveTest. We'll see if it passes the CI

Reply:
Nope, does not work.